metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -704,14 +707,47 @@ def _get_schema(
         ),
     }
 
+    # Add a pattern-based regex for checking if a tag is a group_label tag
+    GROUP_LABEL_TAG_PATTERN = re.compile(r"^looker\.group_label\.(.+)$")
+
     @staticmethod
     def _get_tag_mce_for_urn(tag_urn: str) -> MetadataChangeEvent:
-        assert tag_urn in LookerUtil.tag_definitions
-        return MetadataChangeEvent(
-            proposedSnapshot=TagSnapshotClass(
-                urn=tag_urn, aspects=[LookerUtil.tag_definitions[tag_urn]]
+        # Check if this is a group_label tag
+        tag_name = tag_urn[len("urn:li:tag:") :]
+        match = LookerUtil.GROUP_LABEL_TAG_PATTERN.match(tag_name)
+
+        if match:
+            # This is a group_label tag, create tag definition on the fly
+            group_label_value = match.group(1)
+            tag_properties = TagPropertiesClass(
+                name=f"looker.group_label.{group_label_value}",
```

Comment:
In dbt, we have `:` as separator, instead of `.` https://github.com/search?q=repo%3Adatahub-project%2Fdatahub+%22urn%3Ali%3Atag%3Adbt%22+language%3Ajson&type=code

I followed that approach for hex, https://github.com/search?q=repo%3Adatahub-project%2Fdatahub+%22urn%3Ali%3Atag%3Ahex%22+language%3Ajson&type=code

Not sure if that makes it an standard 😄 


```suggestion
                name=f"looker:group_label:{group_label_value}",
```

Reply:
let me make this change - I originally had it this way, but as the labels do not have a format of `looker:...` I felt that it was probably more something I should keep consistent with the other looker tags. An option would be to make these all consistent - but this would be a breaking change.

Reply:
So, do you mean the Looker source is producing other tags with the pattern `looker.xxx.yyy`?  
If that’s the case, then more important than being consistent with other sources is being consistent with itself 😅

Indeed, I would avoid breaking changes on this.